2023-04-25 02:03:17 [scrapy.utils.log] INFO: Scrapy 2.6.3 started (bot: spider_traffic)
2023-04-25 02:03:17 [scrapy.utils.log] INFO: Versions: lxml 4.6.4.0, libxml2 2.9.5, cssselect 1.1.0, parsel 1.6.0, w3lib 1.22.0, Twisted 22.2.0, Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)], pyOpenSSL 22.0.0 (OpenSSL 1.1.1m  14 Dec 2021), cryptography 36.0.1, Platform Windows-10-10.0.19041-SP0
2023-04-25 02:03:17 [scrapy.crawler] INFO: Overridden settings:
{'BOT_NAME': 'spider_traffic',
 'LOG_FILE': 'logs\\spider_traffic\\crawl_mapData\\471bfaa1e2ca11edb31bdcf50571b4f1.log',
 'NEWSPIDER_MODULE': 'spider_traffic.spiders',
 'SPIDER_MODULES': ['spider_traffic.spiders']}
2023-04-25 02:03:17 [scrapy.utils.log] DEBUG: Using reactor: twisted.internet.selectreactor.SelectReactor
2023-04-25 02:03:17 [scrapy.extensions.telnet] INFO: Telnet Password: c3bd217ec5af6341
2023-04-25 02:03:17 [scrapy.middleware] INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2023-04-25 02:03:17 [twisted] CRITICAL: Unhandled error in Deferred:
2023-04-25 02:03:17 [twisted] CRITICAL: 
Traceback (most recent call last):
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\twisted\internet\defer.py", line 1660, in _inlineCallbacks
    result = current_context.run(gen.send, result)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\scrapy\crawler.py", line 100, in crawl
    self.spider = self._create_spider(*args, **kwargs)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\scrapy\crawler.py", line 112, in _create_spider
    return self.spidercls.from_crawler(self, *args, **kwargs)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\scrapy\spiders\__init__.py", line 48, in from_crawler
    spider = cls(*args, **kwargs)
  File "c:\windows\temp\spider_traffic-1682359354-gmz_40bc.egg\spider_traffic\spiders\crawl_mapData.py", line 21, in __init__
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\io\parsers\readers.py", line 680, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\io\parsers\readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\io\parsers\readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\io\parsers\readers.py", line 1217, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "C:\Users\86166\AppData\Local\Programs\Python\Python310\lib\site-packages\pandas\io\common.py", line 789, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/city_code.csv'
